hi-ml/src/health_ml/utils/lr_scheduler.py

```diff
@@ -0,0 +1,161 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+from typing import Dict, List
+
+from torch.optim.lr_scheduler import CosineAnnealingLR, ExponentialLR, LambdaLR, MultiStepLR, StepLR, _LRScheduler
+from torch.optim.optimizer import Optimizer
+
+from health_ml.deep_learning_config import LRSchedulerType, LRWarmUpType, OptimizerParams
+
```

Comment:
Not sure if this whole file is good to have in hi-ml. It is doing lots of logic that is coupled with the actual models, and it could be that nobody really will use that. Maybe people just prefer to write their own optimizer/LR scheduler without additional support?